plugins/processors/breaker/README.md

```diff
@@ -0,0 +1,74 @@
+# Breaker processor Plugin
+
+This processor could drop metrics based on the value of another selected (flag) metric.
+
+The parameters ``name`` y ``field`` are used to select with metric will be used to define the status of the breaker.
+
+``value_enable`` y ``value_disable`` define which values from the control metric will be used to enable or disable the breaker.
+
+Breaker is "active" means it will block all metrics passing through it.
+
+``enabled`` is the state of the breaker at startup.
+
+The metric used se the "flag" will pass also.
+
+Normally this processor will be used with some filter to select which metrics should be affected.
+
+### Configuration:
+
+```toml
+[[processors.breaker]]
+  # By default is false, meaning it will block
+  enabled = false
+
+  # Select which metric should be used to define the state of the breaker
+  name = "flag_metric"
```

Comment:
Could we use namepass and namedrop instead of adding another way to select the metric?  See docs/CONFIGURATION.md